src/arguments.js

```diff
@@ -89,18 +92,29 @@ const isValueBoolean = (string) => {
 };
 
 /**
- * Checks if argument string has any types by
- * additional chars after property name
+ * Get type of an argument (string, number, null (for flags), etc.)
  * @param {string} string - argument string without description
- * @return {bool} - result of check
+ * @param {object} context - internal parser config
+ * @return {string} argumentType - type of argument
  */
-const isPropertyTyped = (string) => {
-    let isType = false;
+const getPropertyType = (string, context) => {
+    let isFlag = true;
+    let argumentType = 'default';
     string.split(/\s+/).map((argumentSubstring) => {
         argumentSubstring.trim();
-        isType = argumentSubstring.indexOf('-') !== 0 ? true : isType;
+        isFlag = argumentSubstring.indexOf('-') !== 0 ? false : isFlag;
+        if (!isFlag) {
+            Object.keys(context.argumentTypes).forEach((type) => {
```

Comment:
I think we need to create a separate function which identity types of argument, @Arcle1 what do you think?

Reply:
@svoboda-rabstvo done. Please, recheck PR.